### PR TITLE
idc

### PR DESCRIPTION
**Pull request by degenhard**

### CODE REVIEW

**Comment by uzquiano** [comment]
Thank you so much.  Looks really great!

File: src/js/messages/i18n/de_DE.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+	var Alpaca = $.alpaca;
+
+	Alpaca.registerView ({
+		"id": "base",
+		"messages": {
+            "de_DE": {
+                required: "Eingabe erforderlich",
+                invalid: "Eingabe ungültig",
+                months: ["Jänner", "Februar", "März", "April", "Mai", "Juni", "Juli", "August", "September", "Oktober", "November", "Dezember"],
```
Comment (cniesen) [inline src/js/messages/i18n/de_DE.js]: "Jänner" should be "Januar"

Comment (uzquiano) [inline src/js/messages/i18n/de_DE.js]: I don't speak German.  Is Jänner or Januar preferred?

Comment (cniesen) [inline src/js/messages/i18n/de_DE.js]: Januar is the proper German (i.e de_DE).  Jänner is used regionally. I'm not sure if Jänner is the official usage in Austria (i.e. at_DE) but I know for certain it's not in Germany.

**Comment by uzquiano** [comment]
Cool.  Would you be interested in submitting a pull request with adjustments to de_DE?  Let's merge it in. 

@degenhard Do you agree?  Is there a regional variant we should include as well?  We can have a de_<something> if it makes sense.

As I said, I don't speak German so whatever you guys decide is fine by me.

**Comment by cniesen** [comment]
The de_DE dictionary is the primary German dictionaries.  Austrian and Swiss have a few deviations.  So I would definitely keep the de_DE localization.  That said, my current usage of Alpaca doesn't depend on localization and so only English is used. 

BTW, I didn't find Jänner in the hunspell dictionary of Austrian German.
https://raw.githubusercontent.com/elastic/hunspell/master/dicts/de_AT/de-AT.dic

**Comment by cniesen** [comment]
PR #320 created.

**Comment by ApfelUser** [comment]
"Jänner" doesn't exist in the German language :smile: It is swedish. "Januar" is correct.

**Pull request merged**